### DISCUSSION TITLE
[Feature Request] Support Infinite Pagination for V4

### DISCUSSION THREAD

**Discussion by manojhl** [filamentphp/filament#12764] [Ideas]
I was listening to the podcast where danharrin mentioned they have started working on v4 already and planning on parting ways with tailwindcss.

I believe this was the major performance issue for on-click load more or infinite-scroll along with standard Laravel pagination feature.

Let me know what you guys think?

**Comment by Fibtegis** [comment]
Not for v4, but you can use in the v3. 

https://github.com/Fibtegis/filament-infinite-scroll
